manifest_staging/charts/workload-identity-webhook/values.yaml

```diff
@@ -38,3 +38,5 @@ mutatingWebhookNamespaceSelector: {}
 podDisruptionBudget:
   minAvailable: 1
   # maxUnavailable: 0
+
+revisionHistoryLimit: 10
```

Comment:
Instead of setting this here, I would just leave it empty so the kubernetes default value is used (same as old behavior)